utils/netparams.go

```diff
@@ -4,20 +4,21 @@ import (
 	"strings"
 
 	"github.com/decred/dcrd/chaincfg/v2"
+	"github.com/decred/dcrwallet/errors"
 )
 
 var (
 	mainnetParams = chaincfg.MainNetParams()
 	testnetParams = chaincfg.TestNet3Params()
 )
 
-func ChainParams(netType string) *chaincfg.Params {
+func ChainParams(netType string) (*chaincfg.Params, error) {
 	switch strings.ToLower(netType) {
 	case strings.ToLower(mainnetParams.Name):
-		return mainnetParams
+		return mainnetParams, nil
 	case strings.ToLower(testnetParams.Name):
-		return testnetParams
+		return testnetParams, nil
 	default:
-		return nil
+		return nil, errors.New("net type must be either mainnet or testnet3")
```

Comment:
I'd just say "invalid net type" as we can extend this to support other net types in the future. I remember it was discussed once to support simnet (particularly for godcr, I think).